### ISSUE TITLE
pkgconf not found in sle15sp3

### ISSUE THREAD

**Issue by mloviska** [linux-test-project/ltp#870]
Hello, 
seems like there might be a small typo in package name.

In TW's ci script the package is referenced correctly https://github.com/linux-test-project/ltp/blob/master/ci/tumbleweed.sh#L31

https://github.com/linux-test-project/ltp/blob/8638b6e2e110a9cfc90491a23a9b236de00d3c5a/INSTALL#L13

```
15sp3-qu1:~ # zypper se pkgconf pkg-config
S | Name              | Summary                          | Type
--+-------------------+----------------------------------+--------
i | pkg-config        | A library management system      | package
  | python3-pkgconfig | Interface Python with pkg-config | package
 ```

**Comment by pevik** [comment]
TL;DR I'll fix the name in INSTALL, thanks for a report.

FYI, [`pkg-config`](https://gitlab.freedesktop.org/pkg-config/pkg-config) (still somehow live project with [latest commit 8 months ago](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/commits/master) but with [latest release `0.29.2` 4 years ago](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/tags)) was replaced on new distros with more active fork [`pkgconf`](http://pkgconf.org/) (openSUSE [`pkgconf-pkg-config`](https://build.opensuse.org/package/view_file/devel:tools/pkgconf/pkgconf.spec?expand=1), Fedora: https://fedoraproject.org/wiki/Changes/pkgconf_as_system_pkg-config_implementation)

Thus, Tumbleweed has `pkgconf-pkg-config` installed via older `pkg-config`:
```
# zypper install -y pkg-config
Loading repository data...
Reading installed packages...
'pkg-config' not found in package names. Trying capabilities.
'pkgconf-pkg-config' providing 'pkg-config' is already installed.
```

**Comment by mloviska** [comment]
> TL;DR I'll fix the name in INSTALL, thanks for a report.
> 
> FYI, [`pkg-config`](https://gitlab.freedesktop.org/pkg-config/pkg-config) (still somehow live project with [latest commit 8 months ago](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/commits/master) but with [latest release `0.29.2` 4 years ago](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/tags)) was replaced on new distros with more active fork [`pkgconf`](http://pkgconf.org/) (openSUSE [`pkgconf-pkg-config`](https://build.opensuse.org/package/view_file/devel:tools/pkgconf/pkgconf.spec?expand=1), Fedora: https://fedoraproject.org/wiki/Changes/pkgconf_as_system_pkg-config_implementation)
> 
> Thus, Tumbleweed has `pkgconf-pkg-config` installed via older `pkg-config`:
> 
> ```
> # zypper install -y pkg-config
> Loading repository data...
> Reading installed packages...
> 'pkg-config' not found in package names. Trying capabilities.
> 'pkgconf-pkg-config' providing 'pkg-config' is already installed.
> ```

Ah thanks for explaining. I have just noticed that `zypper` yield not found package while copy&paste-ing from INSTALL. :)

**Comment by pevik** [comment]
I usually run `ci/*.sh` script, that's why I haven't noticed. It'd be better to have setup only on single place, but these scripts install also optional dependencies. Once https://github.com/metan-ucw/runltp-ng get's merged into LTP, scripts will be replaced with [`install_pkg.pm`](https://github.com/metan-ucw/runltp-ng/blob/master/install_pkg.pm), thus duplicity will be removed.
